doc/rose-rug-advanced-tutorials-bunch.html

```diff
@@ -0,0 +1,333 @@
+      app under load balancing systems such as PBS or Slurm, you will need to
+      set resource requests to reflect the resources required by running
+      multiple commands at once.</p>
+
+      <p>For example, if a single command would require 1GB memory and the app
+      is configured to run up to 4 commands at once then 4GB of memory should
+      be requested.</p>
+    </div>
+
+    <div class="slide">
+      <h2 id="example-task">Example</h2>
+
+      <p>In this example we are going to create a suite that simulates the
+      handling of landing planes at an airport. For a given plane the process
+      of landing and unloading is the same: land, taxi to the terminal, unload
+      passengers and get clear. We can refer to this as the "landing" routine.
+      What differs between landings is the plane type, number of passengers
+      carried and the resulting timings for each stage of the landing
+      process.</p>
+
+      <p>Create a <a href="rose-rug-brief-tour#cli">new suite</a> (or just a
+      new directory somewhere - e.g. in your homespace) containing a blank
+      <samp>rose-suite.conf</samp> and a <samp>suite.rc</samp> file that looks
+      like this:</p>
+    </div>
+
+    <div class="slide">
+      <h3 class="alwayshidden">Example suite.rc</h3>
+      <pre class="prettyprint lang-cylc">
+[cylc]
+    UTC mode = True # Ignore DST
+    [[event hooks]]
+        timeout handler = rose suite-hook --mail --shutdown
+        timeout = P3D
+[scheduling]
+    [[dependencies]]
+        graph = lander
+[runtime]
+    [[root]]
+        command scripting = rose task-run
+    [[lander]]
+</pre>
+    </div>
+
+    <div class="slide">
+      <h3 class="alwayshidden">Example App</h3>
+
+      <p>In the suite directory create an <samp>app/</samp> directory</p>
+      <pre class="shell">
+mkdir app
+</pre>
+
+      <p>In the <var>app</var> directory create a <var>lander/</var>
+      directory</p>
+      <pre class="shell">
+cd app
+mkdir lander
+</pre>
+    </div>
+
+    <div class="slide">
+      <h3 class="alwayshidden">Example App (2)</h3>
+
+      <p>In the <var>app/lander/</var> directory create a
+      <samp>rose-app.conf</samp> file. Using your editor of choice, paste the
+      following lines into it:</p>
+      <pre class="prettyprint lang-cylc">
+mode=rose_bunch
+
+[bunch]
+command-format=land %(class)s %(passengers)s
+
+[bunch-args]
+class=airbus concorde airbus cessna
+passengers=40 20 30 2
+</pre>
+
+      <p>This configuration will run a <code>rose bunch</code> task that calls
+      multiple instances of the <code>land</code> command, supplying arguments
+      to each instance from the <var>class</var> and <var>passengers</var>
+      entries under <var>[bunch-args]</var>.</p>
+    </div>
+
+    <div class="slide">
+      <h3 class="alwayshidden">Example App (3)</h3>
+
+      <p>In the <var>app/lander/</var> directory create a <samp>bin/</samp>
```

Comment:
Your level of instruction is inconsistent, I would prefer it if you followed the same level of instructions all the way through, if you are going to tell them to `cd` and `mkdir` then you should do it throughout the tutorial.